numpy/_core/tests/test_stringdtype.py

```diff
@@ -1252,6 +1252,27 @@ def test_binary(string_array, unicode_array, function_name, args):
         assert 0
 
 
+@pytest.mark.parametrize("function, expected", [
```

Comment:
These added tests failed on master, because non-python ints were not recognized.